tox.ini

```diff
@@ -48,7 +48,6 @@ passenv =
     dev: CLIENT_EMBED_URL
     dev: SIGNED_URLS_REQUIRED
     dev: YOUTUBE_API_KEY
-    dev: YOUTUBE_PROXY
```

Comment:
I edited this file manually in c141d88464488bf8613a798166c8cb680e216779 without realizing that it is updated by the cookiecutter. Fortunately this change is not that important and we can re-introduce it when needed.